addon/components/sub-form.hbs

```diff
@@ -9,8 +9,33 @@
             </AuHeading>
           {{/if}}
         </Group>
-        {{#if @canRemove}}
+        {{#if (or @canRemove @canMoveUp @canMoveDown)}}
           <Group>
+            <AuButtonGroup @inline={{(and @canMoveUp @canMoveDown)}}>
```

Comment:
Only implemented for the "new listing layout" I'm not sure if the old one needs it. Only LPDC uses that (with some nasty CSS hacks to force fields side-by-side).

Reply:
Added it to the old layout as well.